### ISSUE TITLE
Set up Github Wiki for this repo, with a link from the Readme.

### ISSUE THREAD

**Issue by meelijane** [drupalmel/drupalmel#18]
It could include information about how Blogs are syndicated, how events are syndicated, and how to get your blog or event included on this site.

**Comment by Decipher** [comment]
I'd almost go a step further and suggest utilization of the Github Wiki with a link to it from the Readme.

As mentioned in #6, I'd also like the ability for people to suggest a blog feed, although that will likely need a vetting process to ensure it doesn't get abused, which means we'll need to write some guidelines.

As for Meetups, anyone can suggest a meetup to the group, so that should cover events for the most part.

**Comment by meelijane** [comment]
yeah agreed, that sounds more comprehensive! let's start by setting up the wiki, i'll update this issue.

**Comment by Decipher** [comment]
Wiki front page is partially stubbed. Looking for more people to help. I'm happy to flesh out processes, but not an expert on the best structure for a wiki.
